home.spec.js

```diff
@@ -0,0 +1,30 @@
+var homePage = require('./pages/home');
+
+describe('Mercury Tours demo site', function () {
+
+	beforeEach(function () {
+		homePage.get();
+	});
+
+	it('hould have home page with title Welcome: Mercury Tours', function () {
+		expect(homePage.getTitle()).toEqual("Welcome: Mercury Tours");
+	});
+
+	it('should have footer with © 2005, Mercury Interactive (v. 011003-1.01-058)', function () {
+		expect(homePage.getText()).toEqual("© 2005, Mercury Interactive (v. 011003-1.01-058)");
+	});
+
+	it('should have visable Mercury Tours logo after entering site' , function () {
+		expect(homePage.visableLogo()).toBe(true);
+	});
+
+	it('should have menu items, top and left', function () {
+		expect(homePage.getLeftMenuAtIdx(2).getText()).toBe("Hotels");
```

Comment:
Can you change this test so it verifies all menu items in both menus. Please create a function that returns all menu elements for each menu.